### PR TITLE
Put travis-ci back

### PR DESCRIPTION
**Pull request by Droogans**
Upgrades the visreg client to the latest version, which irons out the kinks in testing both chrome and firefox, which is now done everywhere.

I also enabled source maps in the final js files output by typescript. This lets ava find the original .ts file used to generate the snapshot files.

### LGTM's
- [x] Dev LGTM
- [ ] Zoom LGTM

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "lodash": "^4.17.4",
     "opn": "^5.1.0",
     "selenium-webdriver": "^3.5.0",
-    "snappit-visual-regression": "^0.6.4",
+    "snappit-visual-regression": "0.7.1",
```
Comment (CITguy) [inline test/package.json]: Please remove `selenium-webdriver` from dependencies.

File: test/regression/generateRegressionTests.ts [hunk 2]
```diff
@@ -1,59 +1,61 @@
...
 
 import {test, TestContext} from "ava";
 
 import * as util from "../common/util";
 
-let snappit: Snappit;
-let driver: WebDriver;
-
-const grepCommand = 'grep "\\bdata-visreg=" ../source/components/**/*.html';
+const grepCommand = 'grep "\\bdata-visreg=" ../docs/components/**/*.html ../docs/elements/**/*.html';
```
Comment (CITguy) [inline test/regression/generateRegressionTests.ts]: Maybe `../docs/**/*.html` to simplify?

File: test/regression/generateRegressionTests.ts [hunk 3]
```diff
@@ -1,59 +1,61 @@
...
 
 import {test, TestContext} from "ava";
 
 import * as util from "../common/util";
 
-let snappit: Snappit;
-let driver: WebDriver;
-
-const grepCommand = 'grep "\\bdata-visreg=" ../source/components/**/*.html';
+const grepCommand = 'grep "\\bdata-visreg=" ../docs/**/**/*.html';
```
Comment (CITguy) [inline test/regression/generateRegressionTests.ts]: Will this not work with a single set of double asterisks? `../docs/**/*.html` vs `../docs/**/**/*.html`?

Comment (Droogans) [reply test/regression/generateRegressionTests.ts]: I can use

```
grep "\\bdata-visreg=" -r ../docs/
```

But as far as I know, I can't scope it to just html files without forcing it to glob through sub directories manually.

Comment (Droogans) [reply test/regression/generateRegressionTests.ts]:
```
grep "\\bdata-visreg=" -r ../docs | grep "index.html" 
```

would do it though.

**Pull request merged**